src/app/components/core/user-management/create-user/create-user.component.html

```diff
@@ -34,36 +42,73 @@
           </div>
           <div class="columns">
             <div class="column">
+              <label class="label is-small">Authentication Method</label>
+              <div id="access-method-dropdown" class="dropdown">
+                  <div class="dropdown-trigger">
+                    <button type="button" class="button is-fullwidth is-small" aria-haspopup="true"
+                      aria-controls="dropdown-menu" (click)="toggleDropDown('access-method-dropdown')">
+                      <span>{{selectedAuthMethod}}</span>
+                      <span class="icon is-small">
+                        <i class="fa fa-sm fa-angle-down" aria-hidden="true"></i>
+                      </span>
+                    </button>
+                  </div>
+                  <div class="dropdown-menu" id="dropdown-menu" role="menu">
+                    <div class="dropdown-content">
+                      <a class="dropdown-item" *ngFor="let loginMethod of loginMethods"
+                        (click)="toggleDropDown('access-method-dropdown');setAccessMethod(loginMethod)">
+                        {{loginMethod.text}}
```

Comment:
loginMethod, selectedAuthMethod setAccessMethod please be consistent with terminology along with display